inc/namespace.php

```diff
@@ -501,6 +501,11 @@ function load_plugins() {
 	if ( $config['healthcheck'] ) {
 		add_action( 'plugins_loaded', __NAMESPACE__ . '\\load_healthcheck' );
 	}
+
+	// Bootstrap integration with Elasticsearch Service API.
+	if ( Altis\get_config()['modules']['search']['enabled'] ?? false ) {
```

Comment:
```suggestion
	if ( ! empty( Altis\get_config()['modules']['search']['enabled'] ) ) {
```

IMO this is more readable / simpler than the null-coalesce.

Reply:
I've got it in my head that it's safer to do it using null coalesce because `empty()` doesn't check each level of the array - I'm conflating it with `isset()`!